### ISSUE TITLE
GTA V startup crashes

### ISSUE THREAD

**Issue by sevenleo** [nefarius/ScpToolkit#61]
GTA V startup crashes if scpdriver is enabled, is necessary disable, start the game, after the game is ful load press ALT+TAB and start scpdriver.
Why?
All the games i've tested worked fine, but this (my favorite) not.

**Comment by nefarius** [comment]
I now own GTA V too so I can test this myself in a few days hopefully.

**Comment by The-Euphoria** [comment]
@nefarius - Is there any update on this??? As my report is also regarding GTA V. My report: https://github.com/nefarius/ScpToolkit/issues/311

**Comment by The-Euphoria** [comment]
Never mind, found the cause of the issue and the fix :)

**Comment by nefarius** [comment]
@The-Euphoria mind to share the solution?

**Comment by The-Euphoria** [comment]
@nefarius - Disable 'Vibration' in-game, that's all it was.
I read a lot of posts of people having issue with the vibrate/rumble in some games, so I disabled vibration in the in-game menu and Bingo, no more issues, re-enable vibration and instant CTD. I got to speaking with someone afterwards, he said he'd discovered the current Scp driver has an issue with vibration in GTA and some other games causing it to freeze all input while vibrating, or completely crash the driver causing game crashes and even BSOD, which effected me multiple times.

Hope this helps
